db/sg_replicate_cfg.go

```diff
@@ -107,6 +107,7 @@ type ReplicationConfig struct {
 	Cancel                 bool                      `json:"cancel,omitempty"`
 	Adhoc                  bool                      `json:"adhoc,omitempty"`
 	BatchSize              int                       `json:"batch_size,omitempty"`
+	RunAsAdminUser         bool                      `json:"run_as_admin_user,omitempty"`
```

Comment:
Yes we discussed it, but iirc, we where going to keep it unseparated, and separate it in the future if it gets confusing.

Reply:
I had the impression of the reverse coming away from those discussions :) It was already confusing having them mixed, and to do it correctly now rather than later and then have to worry about backwards compatibility.

Reply:
If you're unsure, let's leave it for now and decide on a call